python/ray/data/dataset.py

```diff
@@ -1645,11 +1645,14 @@ def __iter__(self):
         return DatasetPipeline(Iterable(self), length=times)
 
     def pipeline(self, *, parallelism: int = 10) -> "DatasetPipeline[T]":
-        """Pipeline the dataset execution by splitting its blocks into groups.
+        raise DeprecationWarning("Use .window(n) instead of .pipeline(n)")
```

Comment:
Given that we're still alpha, shouldn't we just delete this method?

Reply:
Maybe, but it doesn't hurt much to leave it for a bit.